### PR TITLE
Feature/271 quantile

### PR DESCRIPTION
**Pull request by johcarter**
#271 

### Added quantile report tests
Added tests for QELT and QPLT reports

### CODE REVIEW

File: ktest/runtests.sh [hunk 1]
```diff
@@ -73,6 +73,18 @@ installertest()
...
+	../src/eltcalc/eltcalc -Q ../ktest/testout/gulqelt2.csv < ../ktest/testout/gulsummarycalc2.bin 
+	#../src/eltcalc/eltcalc -Q ../ktest/testout/fmmelt1.csv < ../ktest/testout/fmsummarycalc1.bin 
+	#../src/eltcalc/eltcalc -Q ../ktest/testout/fmmelt2.csv < ../ktest/testout/fmsummarycalc2.bin 
+
+	# test qplt
+	../src/pltcalc/pltcalc -Q ../ktest/testout/gulqplt1.csv < ../ktest/testout/gulsummarycalc1.bin 
+	../src/pltcalc/pltcalc -Q ../ktest/testout/gulqplt2.csv < ../ktest/testout/gulsummarycalc2.bin 
+	#../src/pltcalc/pltcalc -Q ../ktest/testout/fmmplt1.csv < ../ktest/testout/fmsummarycalc1.bin 
+	#../src/pltcalc/pltcalc -Q ../ktest/testout/fmmplt2.csv < ../ktest/testout/fmsummarycalc2.bin
+
```
Comment (hchagani-oasislmf) [inline ktest/runtests.sh:87]: It is possible to output the QELT and MELT (or QPLT and MPLT) in a single run of `eltcalc` (or `pltcalc`). So, the following would also be valid:

```
../src/eltcalc/eltcalc -M ../ktest/testout/gulmelt1.csv -Q ../ktest/testout/gulqelt1.csv < ../ktest/testout/gulsummarycalc1.bin
../src/pltcalc/pltcalc -M ../ktest/testout/gulmplt1.csv -Q ../ktest/testout/gulqplt1.csv < ../ktest/testout/gulsummarycalc1.bin
```

**Review by hchagani-oasislmf — APPROVED** [review]
Looks good. Just one comment on an optional change.

File: ktest/runtests.sh (hunk 1 above)
Comment (johcarter) [reply ktest/runtests.sh:87]: sure we can make that change

**Review by hchagani-oasislmf — APPROVED** [review]
Thanks. Looks good.

**Pull request merged**